content/en/docs/ambient/usage/l4-policy/index.md

```diff
@@ -0,0 +1,88 @@
+
+The Layer 4 (L4) features of Istio's [security policies](/docs/concepts/security) are supported by ztunnel, and are available in ambient mode. [Kubernetes Network Policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/) also continue to work if your cluster has a {{< gloss >}}CNI{{< /gloss >}} plugin that supports them, and can be used to provide defense-in-depth.
+
+To use L7 policies, and Istio's traffic routing features, you can [deploy a waypoint](/docs/ambient/usage/waypoint) for your workloads.
+
+## Layer 4 authorization policies
+
+The ztunnel proxy performs authorization policy enforcement when a workload is enrolled in secure overlay mode (i.e. with no waypoint proxy configured).
+
+The actual enforcement point is at the receiving (server-side) ztunnel proxy in the path of a connection.
+
+A basic L4 authorization policy looks like this:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1
+kind: AuthorizationPolicy
+metadata:
+ name: allow-sleep-to-httpbin
+spec:
+ selector:
+   matchLabels:
+     app: httpbin
+ action: ALLOW
+ rules:
+ - from:
+   - source:
+       principals:
+       - cluster.local/ns/ambient-demo/sa/sleep
+EOF
+{{< /text >}}
+
+The behavior of the `AuthorizationPolicy` API has the same functional behavior in Istio ambient mode as in sidecar mode. When there is no `AuthorizationPolicy` provisioned, then the default action is `ALLOW`. Once a policy is provisioned, pods matching the selector in the policy only allow traffic which is explicitly allowed. In this example, pods with the label `app: httpbin` only allow traffic from sources with an identity principal of `cluster.local/ns/ambient-demo/sa/sleep`. Traffic from all other sources will be denied.
+
+### Layer 7 authorization policies without waypoints installed
+
+{{< warning >}}
+If an `AuthorizationPolicy` has been configured that requires any traffic processing beyond L4, and if no waypoint proxies are configured for the destination of the traffic, then ztunnel proxy will simply drop all traffic as a defensive move. Hence, check to ensure that either all rules involve L4 processing only or else if non-L4 rules are unavoidable, that waypoint proxies are configured.
+{{< /warning >}}
+
+This example adds a check for the HTTP GET method:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+ name: allow-sleep-to-httpbin
+spec:
+ selector:
+   matchLabels:
+     app: httpbin
+ action: ALLOW
+ rules:
+ - from:
+   - source:
+       principals:
+       - cluster.local/ns/ambient-demo/sa/sleep
+   to:
+   - operation:
+       methods: ["GET"]
+EOF
+{{< /text >}}
+
+Even though the identity of the pod is otherwise correct, the presence of a L7 policy causes the ztunnel to deny the connection:
```

Comment:
```suggestion
Even though the identity of the pod is otherwise correct, the presence of a L7 policy plus the traffic isn't from waypoint causes the ztunnel to deny the connection:
```